security.md

```diff
@@ -10,13 +10,11 @@ Access to the Web Admin is protected via Azure AD authentication using OIDC with
 
 ### Authenticating with the tenant
 
-Simeon can use service principal authentication, delegated authentication and/or a service account to interact with your tenants.
-
-If you use service principal authentication, you must still use either delegated authentication or service account authentication in conjuction in order for Simeon to connect to certain Azure AD and Teams APIs that do not support service principal authentication.
+ By default, Simeon uses a service principal to authenticate into in the tenant and manage supported configurations. A user account, either delegated authentication or a service account, is still required for Simeon to connect to certain Azure AD and Teams APIs that do not support service principal authentication.
```

Comment:
A user account, authenticating with device code or password authentication, is still required....

Reply:
Made the change